docs/index.rst

```diff
@@ -0,0 +1,33 @@
+Monitoring
+==========
+
+OpenWISP Monitoring is a network monitoring system written in Python and
+Django, designed to be **extensible**, **programmable**, **scalable** and
+easy to use by end users: once the system is configured, monitoring
+checks, alerts and metric collection happens automatically.
```

Comment:
I made Google Gemini write this. This is placeholder for now.

Reply:
What about:

The OpenWISP Monitoring module leverages the capabilities of Python and the Django Framework to provide OpenWISP with robust network monitoring features. Designed to be extensible, programmable, scalable, and user-friendly, this module automates monitoring checks, alerts, and metric collection, ensuring efficient and comprehensive network management.